code/data_preprocessing/reference_data.ipynb

```diff
@@ -782,13 +782,44 @@
     "        f\"{cwd}/RSEM_Index/rsem_reference.chrlist\", f\"{cwd}/RSEM_Index/rsem_reference.seq\", \n",
     "        f\"{cwd}/RSEM_Index/rsem_reference.transcripts.fa\"\n",
     "task: trunk_workers = 1, trunk_size = job_size, walltime = walltime, mem = mem, tags = f'{step_name}_{_output[0]:bd}'\n",
-    "bash: container=container, expand= \"${ }\", stderr = f'{_output[0]:d}.stderr', stdout = f'{_output[0]:d}.stdout'\n",
+    "bash: container=container, expand= \"${ }\", stderr = f'{_output[0]}.stderr', stdout = f'{_output[0]}.stdout'\n",
```

Comment:
same comment here: we should swap the output so that we have rsem_reference.stdout. I'll do that after we merge it.

Reply:
Actually there are a couple more items @hsun3163 you need to fix. I'll therefore leave this to you: please swap `rsem_reference.grp` to _output[0] so we can have `rsem_reference.stdout`. Notice that you need to double check the commands to make sure eg you use _output[0] instead of _output[1]